src/applications/vaos/actions/appointments.js

```diff
@@ -0,0 +1,21 @@
+import pending from './requests.json';
+
+export const FETCH_PENDING_APPOINTMENTS = 'vaos/FETCH_PENDING_APPOINTMENTS';
```

Comment:
Just set this up so that I could have some placeholder Redux code. This will get adjusted to what @narin writes in his ticket.